site/source/user-manual/service-guides/bitcoin/bitcoin-integrations.rst

```diff
@@ -146,3 +146,22 @@ Trezor Suite
  **Instructions**
   
   Follow the `guide <https://github.com/Start9Labs/electrs-wrapper/blob/master/docs/integrations/trezor/guide.md>`__.
+  
```

Comment:
One nitpick - these are in alphabetical order - please move up above Samourai and we'll get this merged

Reply:
updated